### ISSUE TITLE
Tests fail to compile: fatal error: 'faiss/perf_tests/utils.h' file not found

### ISSUE THREAD

**Issue by yurivict** [facebookresearch/faiss#3913]
# Summary

```
/usr/ports/math/faiss/work/faiss-1.9.0/perf_tests/utils.cpp:1:10: fatal error: 'faiss/perf_tests/utils.h' file not found
    1 | #include <faiss/perf_tests/utils.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

# Platform

OS: FreeBSD 14.1

Faiss version: 1.9.0

Faiss compilation options:
-DCMAKE_C_COMPILER:STRING="cc"  -DCMAKE_CXX_COMPILER:STRING="c++"  -DCMAKE_C_FLAGS:STRING="-O2 -pipe  -fstack-protector-strong -isystem /usr/local/include -fno-strict-aliasing "  -DCMAKE_C_FLAGS_DEBUG:STRING="-O2 -pipe  -fstack-protector-strong -isystem /usr/local/include -fno-strict-aliasing "  -DCMAKE_C_FLAGS_RELEASE:STRING="-O2 -pipe  -fstack-protector-strong -isystem /usr/local/include -fno-strict-aliasing  -DNDEBUG"  -DCMAKE_CXX_FLAGS:STRING="-O2 -pipe -fstack-protector-strong -isystem /usr/local/include -fno-strict-aliasing  -isystem /usr/local/include "  -DCMAKE_CXX_FLAGS_DEBUG:STRING="-O2 -pipe -fstack-protector-strong -isystem /usr/local/include -fno-strict-aliasing  -isystem /usr/local/include "  -DCMAKE_CXX_FLAGS_RELEASE:STRING="-O2 -pipe -fstack-protector-strong -isystem /usr/local/include -fno-strict-aliasing  -isystem /usr/local/include  -DNDEBUG"  -DCMAKE_EXE_LINKER_FLAGS:STRING=" -fstack-protector-strong -L/usr/local/lib "  -DCMAKE_MODULE_LINKER_FLAGS:STRING=" -fstack-protector-strong -L/usr/local/lib "  -DCMAKE_SHARED_LINKER_FLAGS:STRING=" -fstack-protector-strong -L/usr/local/lib "  -DCMAKE_INSTALL_PREFIX:PATH="/usr/local"  -DCMAKE_AUTOGEN_PARALLEL:STRING="7"  -DCMAKE_BUILD_TYPE:STRING="Release"  -DTHREADS_HAVE_PTHREAD_ARG:BOOL=YES  -DCMAKE_INSTALL_RPATH_USE_LINK_PATH=YES  -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON -DBUILD_SHARED_LIBS:BOOL=ON -DBUILD_TESTING:BOOL=OFF -DFAISS_ENABLE_GPU:BOOL=OFF -DFAISS_ENABLE_PYTHON:BOOL=OFF -GNinja

Running on:
- [ ] CPU
- [ ] GPU

Interface: 
- [x] C++
- [ ] Python

**Comment by ramilbakhshyiev** [comment]
@yurivict Have you tried checking out the repository to `faiss` directory instead of `faiss-1.9.0`?

**Comment by yurivict** [comment]
> Have you tried checking out the repository to faiss directory instead of faiss-1.9.0?

No, because I need to update the FreeBSD port to the latest version.

**Comment by ramilbakhshyiev** [comment]
We don't officially support FreeBSD so trying to see how I could help. My question above was about the directory name not the version you're trying to compile.

**Comment by ramilbakhshyiev** [comment]
Could I ask you to follow the existing CI builds and finding the differences that you could fix for? It seems to find that file correctly in those builds so wonder if that would help you troubleshoot it.

**Comment by yurivict** [comment]
It of course works when faiss is checked out as "faiss" vs "faiss-{version}". However, it is highly unusual to require such check out because tarballs typically extract as "faiss-{version}".

**Comment by alibeklfc** [comment]
#4792 has resolved it. If your issue has not been resolved, please reopen this issue
